### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by Daphne-CPAdmin** [Mac561/compass#1]
Missing Milestone Deliverables
- Gifs created to show build progress and added to README *(3pts)*s

**Comment by codepathreview** [comment]
Missing Milestone Deliverables (No updates, just submitted)

Updated status of issues in Project board (2pts)
Sprint planned for next week (Issues created, assigned & added to project board) (3pts)
Completed user stories checked-off in README (2pts)
Gifs created to show build progress and added to README (3pts) 
/cc @codepathreview

**Comment by Mac561** [comment]
I have all that completed, can I just update it in the morning? Is that
okay??

**Comment by Daphne-CPAdmin** [comment]
Yeah definitely and please let me know as soon as you have updated it.
Thank you!

---

Daphne Jane Rendora
iOS Operations
CodePath.org <http://codepath.org/>
daphne@codepath.org

ᐧ
